index.html

```diff
@@ -154,10 +154,13 @@ <h2 class="contactme">Contact me</h2>
         </p>
       </div>
       <div  class="center">
-        <input type="text" placeholder="Yeremias"><br>
-        <input type="text" placeholder="natajayanj14@gmail.com"><br>
-        <textarea placeholder="Write your message here" name="message" id="message" cols="30" rows="10"></textarea><br>
-        <button  class="button ">Get in Touch</button>
+        <form>
+          <div id="error"></div>
```

Comment:
- [ ] Hey there! 🖐 Just a friendly suggestion - it would be great if the error message 🚨 could be placed a little closer to the submit button 🔘. This way, users will have a smoother experience 🌟. Thanks! 🙌